website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -270,7 +270,7 @@ Karpenter will automatically query for the appropriate [EKS optimized AMI](https
 
 ### Custom
 
-The `Custom` AMIFamily ships without any default userData to allow you to configure custom bootstrapping for control planes or images that don't support the default methods from the other families.
+The `Custom` AMIFamily ships without any default userData to allow you to configure custom bootstrapping for control planes or images that don't support the default methods from the other families. It is required that custom user data adds `--register-with-taints=karpenter.sh/unregistered=true:NoExecute`. Karpenter will fail to register nodes that do not have this taint.
```

Comment:
Can we specify that the taint needs to be added by the kubelet, and link out to a reference to the kubelet flags ([docs](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#options)) and the KubeletConfiguration spec ([docs](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration)). Our default recommendation should also probably be to do this through the kubelet config file since the flag is deprecated.

Reply:
Good callout. Adding now.